backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/DemoOktaRepository.java

```diff
@@ -344,6 +350,37 @@ public Optional<OrganizationRoleClaims> getOrganizationRoleClaimsForUser(String
     }
   }
 
+  public PartialOktaUser findUser(String username) {
+    UserStatus status =
+        inactiveUsernames.contains(username) ? UserStatus.DEPROVISIONED : UserStatus.ACTIVE;
+    boolean isAdmin = adminGroupMemberSet.contains(username);
+
+    Optional<OrganizationRoleClaims> orgClaims;
+
+    try {
+      orgClaims = Optional.ofNullable(usernameOrgRolesMap.get(username));
+    } catch (ScopeNotActiveException e) {
+      // Tests are set up with a full SecurityContextHolder and should not rely on
+      // usernameOrgRolesMap as the source of truth.
+      if (!("UNITTEST".equals(environment)) && usernameOrgRolesMap.containsKey(username)) {
```

Comment:
Non-blocking, maybe addressable in a followup PR: I would love it if we we pulled this string config out into a variable. Worried about potential env drift hardcoding the name like this.

Doing a command+f across the codebase shows me there's quite a few places where we're hardcoding this string, so we might want to do it in a follow up PR. If folks think similarly, I'm happy to put a PR out that pulls those things together

Reply:
Yes @fzhao99! it would be awesome if you can make a tech debt ticket for this. 🙇‍♀️